### PR TITLE
Removed unused Domain attributes

### PR DESCRIPTION
**Pull request by orangejenny**
## Technical Summary
Removes two attributes that aren't referenced anywhere.

This will reindex the ES domains index. Verified that takes 5 minutes on prod, so it should be fine to do in a deploy.

### Safety story
Removal of dead code. Grepped for these attributes in the repo.

### Automated test coverage

None

### QA Plan

No QA

### Rollback instructions

- [x] This PR can be reverted after deploy with no further considerations

### Labels & Review
- [x] Risk label is set correctly
- [x] The set of people pinged as reviewers is appropriate for the level of risk of the change

### CODE REVIEW

**Review by snopoke — COMMENTED** [review]
This won't actually trigger a reindex unless you rename the index name.

Did you try run `./manage.py update_es_mapping.py hqdomain`? I think it will fail since you're removing field from the mapping. Although this won't break deploy it's not great to leave the mapping in a bad state so I think doing a reindex is probably worth it if it's going to be so quick.

**Comment by orangejenny** [comment]
@snopoke Thanks, I forgot about updating the index name. Added b7ffd23ebe3b445e1f3e266ea8844d46af24046c to do so. Prior to the rename, `./manage.py update_es_mapping hqdomains_2020-02-10` did run without erroring (I didn't do until you suggested it, but it did work.) Also, I should have renamed the index in https://github.com/dimagi/commcare-hq/pull/31211/ as well, since that also changes the domain mapping. That doesn't need to be updated since this PR is already renaming the index, right?

@joxl The 5 minutes number is the run time of `django_manage ptop_reindexer_v2 domain` on prod. That does the same reindex operation that'll happen here, doesn't it?

**Comment by joxl** [comment]
> That does the same reindex operation that'll happen here, doesn't it?

I've seen that command, but I don't know what's different between it, other existing reindex commands, and/or if something separate happens "automatically" when pillows launches. I was under the impression that reindexing was a manual process with several steps. I'm not sure how any automatic or single-manage-command operation can guarantee integrity of index data when switching to a new index (i.e. one "cloned" from the old one via the reindex API).

**Review by joxl — APPROVED** [review]

File: corehq/pillows/mappings/domain_mapping.py [hunk 1]
```diff
@@ -1,7 +1,7 @@
 from corehq.util.elastic import prefix_for_tests
 from pillowtop.es_utils import ElasticsearchIndexInfo, DOMAIN_HQ_INDEX_NAME
 
-DOMAIN_INDEX = prefix_for_tests("hqdomains_2020-02-10")
+DOMAIN_INDEX = prefix_for_tests("hqdomains_2021-03-08")
```
Comment (joxl) [inline corehq/pillows/mappings/domain_mapping.py:4]: Just realized this. Did you mean to use 2021, or should this be `2022-03-08`?

Comment (orangejenny) [reply corehq/pillows/mappings/domain_mapping.py:4]: I did mean to use 2022, but it looks like I was consistently wrong throughout the PR, so it shouldn't matter.

Comment (joxl) [reply corehq/pillows/mappings/domain_mapping.py:4]: No need to change it, you've satisfied my curiosity :)

**Comment by joxl** [comment]
It looks like the old index (`hqdomains_2020-02-10`) never got cleaned up. Is there a trigger/process to ensure this is completed after a reindex?

**Comment by joxl** [comment]
It looks like the old index (`hqdomains_2020-02-10`) never got cleaned up. Is there a trigger/process to ensure this is completed after a reindex?

**Pull request merged**